src/NodaTime/Calendars/GJYearMonthDayCalculator.cs

```diff
@@ -9,21 +9,21 @@ namespace NodaTime.Calendars
     internal abstract class GJYearMonthDayCalculator : RegularYearMonthDayCalculator
     {
         // These arrays are NOT public. We trust ourselves not to alter the array.
-        // They use zero-based array indexes so the that valid range of months is
-        // automatically checked. They are protected so that GregorianYearMonthDayCalculator can
-        // read them.
-        protected static readonly int[] MinDaysPerMonth = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
-        protected static readonly int[] MaxDaysPerMonth = { 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
+        // They are protected so that GregorianYearMonthDayCalculator can read them.
+        // The arrays are 1-based (so "days in January" are accessed via array[1]); the index should be validated before
+        // use.
+        private protected static readonly int[] NonLeapDaysPerMonth = { 0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
+        private protected static readonly int[] LeapDaysPerMonth = { 0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
 
         // Note: these fields must be declared after MinDaysPerMonth and MaxDaysPerMonth so that the initialization
```

Comment:
Update comment here, since you've renamed the fields.

Reply:
Thanks, will do that then merge on green.